drivers/serial/Kconfig.nrfx

```diff
@@ -10,6 +10,7 @@ menuconfig UART_NRFX
 	default y
 	select SERIAL_HAS_DRIVER
 	select SERIAL_SUPPORT_INTERRUPT
+	select UART_0_ASYNC if UART_ASYNC_API
```

Comment:
There should be possibility to use UART_0 with interrupt driven API, while other ports use asynchronous API. This line is not needed.

Reply:
You are right. I overlooked that one. Okay, I'll remove all changes in Kconfig. However, I think UART_0_ASYNC should also depend on "!UART_INTERRUPT_DRIVEN" otherwise, both parts could be enabled if nRF UART driver doesn't support that. Also, it looks like the current nRF UART driver only supports UART0.